static/locales/en/translation.json

```diff
@@ -0,0 +1,70 @@
+    "E005": "You need to put the value of the __attribute__ attribute in quotation marks\nTry: __attribute__=\"myvalue\"",
+    "E006": "Every attribute needs a value.\nTry: myattribute=\"myvalue\"",
+    "E007": "The first line of your HTML should always be:\n<!DOCTYPE html>",
+    "E008": "The first line of your HTML should always be:\n<!DOCTYPE html>",
+    "E012": "You can't use the id \"__id__\" more than once in your HTML",
+    "E014": "<img> tags need a src attribute, with the URL of the image you want to display.\nTry: <img src=\"http://coolimages.com/image.jpg\">",
+    "E016": {
+      "b": "You shouldn't use the <b> tag. Use the <strong> tag instead",
+      "big": "You shouldn't use the <big> tag. Use the CSS font-size property instead",
+      "center": "You shouldn't use the <center> tag. Use the CSS property margin: auto instead",
+      "font": "You shouldn't use the <font> tag. Use the CSS font-face property instead.",
+      "i": "You shouldn't use the <i> tag. Use the <em> tag instead",
+      "strike": "You shouldn't use the <strike> tag. Use the CSS property text-decoration: line-through instead",
+      "tt": "You shouldn't use the <tt> tag. Use the <code> tag instead."
+    },
+    "E017": "Use lower case for tag names",
+    "E027": "Put a <title> tag inside your <head> tag",
+    "E028": "You have more than one <title> tag in your document; you should only have one",
+    "E030": "You have an opening tag somewhere in the HTML that doesn't have a closing tag to match it.",
+    "E036": "Lines should be indented with four spaces.\nUse the 'tab' key to increase indentation and the 'delete' key to decrease indentation."
+  },
+  "javascript": {
+    "E019": "There is a __error_a__ on this line that needs a closing __error_a_match__",
+    "E020": "There is a __error_b__ on this line that needs a closing __error_a__",
+    "E030": "There's something missing from this line. Did you forget something?",
+    "W003": "You can't use the variable __error_a__ before declaring it.",
+    "W030": "It looks like there's something on this line that doesn't belong there",
+    "W031": "If you are making a new object, be sure to use it or store it in a variable",
+    "W032": "You don't need a semicolon ; on this line",
+    "W033": "You need a semicolon ; at the end of this line",
+    "W058": "There should be parentheses here.\nTry: new __error_a__()",
+    "W084": "Use === to check if two things are equal",
+    "W098": "You don't use the variable __error_a__ anywhere",
+    "W112": "The string on this line needs to end with another quotation mark",
+    "W116": {
+      "case-1": "You should use === instead of ==",
+      "case-2": "You should use !== instead of !=",
+      "case-3": "There should be a __error_a__ where you wrote __error_b__"
+    },
+    "W117": "You haven't declared __error_a__.\nTry: var __error_a__;",
+    "W123": "You already declared __error_a__ somewhere else."
+  }
+}
```

Comment:
There’s a small handful of other places where English copy is hard-coded in, particularly in [`errorList.jsx`](https://github.com/outoftime/learnpad/blob/master/src/components/errorList.jsx#L28)—would be great to get those into translations as well (totally down to do that in a separate PR if you prefer tho)